gensim/models/word2vec.py

```diff
@@ -1610,13 +1612,16 @@ def accuracy(self, questions, restrict_vocab=30000, most_similar=most_similar, u
                     logger.debug("skipping line #%i with OOV words: %s" % (line_no, line.strip()))
                     continue
 
+                original_vocab = self.vocab
+                self.vocab = ok_vocab
                 ignore = set(self.vocab[v].index for v in [a, b, c])  # indexes of words to ignore
                 predicted = None
                 # find the most likely prediction, ignoring OOV words and input words
                 sims = most_similar(self, positive=[b, c], negative=[a], topn=False, restrict_vocab=restrict_vocab)
+                self.vocab = original_vocab
                 for index in matutils.argsort(sims, reverse=True):
```

Comment:
Yes, that seems correct. 
Just making sure I understand - in case `sort_vocab` is `True` (which it is, by default), using `index2word` to build `ok_vocab` would imitate `compute_accuracy.c`'s behaviour, since the most frequent version of the analogy words would be taken.

In case `sort_vocab` is `False`, the last-seen version of the analogy words would be taken, therefore not quite imitating `compute_accuracy`'s behaviour.